Helper/OmiseHelper.php

```diff
@@ -111,4 +111,13 @@ public function convertTescoSVGCodeToHTML($svg)
         }
         return $xhtml->saveXML(null, LIBXML_NOEMPTYTAG);
     }
+    
+    /**
+     * Checks if $paymentType is offline payment. This method returns true for offline payment method or else false.
+     * @param string $paymentType
+     * @return boolean
+     */
+    public function isOfflinePayment($paymentType) {
+        return ($paymentType === 'paynow' || $paymentType === 'promptpay' || $paymentType === 'bill_payment_tesco_lotus');
+    }
```

Comment:
Understand that this method is designed to check for those QR code payment method. However, the name `isOfflinePayment` gives me an impression that the Konbini payment method should be listed here too.

(alternative suggestion, I think `Model/Config/Promptpay.php` would be more appropriate place to have this kind of configuration instead of putting it in a helper class)

Reply:
Updated method name to be more descriptive. Moving configuration to `Model/Config/Promptpay.php` might won't serve the same purpose. Addressed comment here - https://github.com/omise/omise-magento/pull/264/commits/0d80755685cd8740fc0571a8a94becb41977fac2